test/unit/effects.js

```diff
@@ -17,15 +17,32 @@ QUnit.module( "effects", {
 		this.sandbox = sinon.sandbox.create();
 		this.clock = this.sandbox.useFakeTimers( 505877050 );
 		this._oldInterval = jQuery.fx.interval;
+		this._oldPerformanceNow = window.performance.now;
 		jQuery.fx.step = {};
 		jQuery.fx.interval = 10;
+
+		// Fake performance.now() returning lower values than Date.now()
+		// and that its values are fractional.
+		// Support: IE <10, Safari <8.0, iOS <9, Android <4.4, Node with jsdom 9.4
+		if ( window.performance && typeof window.performance.now === "function" ) {
+			window.performance.now = function() {
+				return Date.now() - 99999.6394;
+			};
+		}
+
 		jQuery.now = Date.now;
```

Comment:
It turns out it was useful as `jQuery.now` is defined as `Date.now` so if you mock `Date.now` `jQuery.now` would remain unmocked.

This is yet another manifestation of how broken our current mocking that happens after jQuery has already loaded & run really is. :/